### PR TITLE
Interview process guide

### PR DESCRIPTION
**Pull request by Geodun**
adding the community spotlight interviewing process guide to the contribution section of the website

### CODE REVIEW

File: docs/guide/index.rst [hunk 1]
```diff
@@ -124,6 +124,7 @@ Additional Notes
 
    about/index
    contributing
+   community-spotlight-interviewing-guide
```
Comment (ericholscher) [inline docs/guide/index.rst]: This should go in the `organizers-guide` file. This is for staff not users.

File: docs/guide/community-spotlight-interviewing-guide.rst [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+Formatting and other extra considerations
+-----------------------------------------
+
+Every interview is written in RST. For an example of what the RST formatting should look like, you can reference `this example`_ or any of the other documents in the community-spotlight directory of the repository.
+
+Companies and well known technologies or tools should link out to relevant web pages. Company links should redirect to their careers page, if available. Technology and tools links should redirect to the standard resource for learning that technology or the WTD page if it exists.
+
+General questions resource
+==========================
```
Comment (ericholscher) [inline docs/guide/community-spotlight-interviewing-guide.rst]: This shouldn't be a top-level `===`, it should be the 2nd level

File: docs/guide/community-spotlight-interviewing-guide.rst [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+    What should an aspiring tech writer be able to do before searching for a job?
+
+    How does the reality of your job differ from the expectations you had before becoming a 
+tech writer?
+
+What motivated you to pursue tech writing rather than a related field?(advertising, 
+journalism, STEM)
+
+Conference speaker interview recommendations
+============================================
```
Comment (ericholscher) [inline docs/guide/community-spotlight-interviewing-guide.rst]: This shouldn't be a top-level `===`, it should be the 2nd level

File: docs/guide/community-spotlight-interviewing-guide.rst [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+Where have you learned the skills that you use for your work?
+
+**Work**
+    
+    What is the story of your career so far?
+
+    What other jobs have you had?
+        How has that experience influenced your work now?    
+
+    How long did it take between learning you wanted to become a tech writer and getting
```
Comment (ericholscher) [inline docs/guide/community-spotlight-interviewing-guide.rst]: Some weird formatting here. We should probably just make these actual bullet points (`*`) indented properly, so they render as a list.

**Review by ericholscher — COMMENTED** [review]
This is great! Just a few cleanup notes.

File: docs/guide/index.rst (hunk 1 above)
Comment (Geodun) [reply docs/guide/index.rst]: Does the file need to go into a folder before I create the toctree, and what do you think the depth should be?

File: docs/organizer-guide/index.rst [hunk 5]
```diff
@@ -68,6 +68,16 @@ Information about our regional Write the Docs conferences.
...
 
+Community Spotlight
+-------------------
+
+Information about contributing interview content for the community.
+
+.. toctree::
+   :maxdepth: 2
+
+   organizer-guide/community-spotlight-interviewing-guide
```
Comment (ericholscher) [inline docs/organizer-guide/index.rst]:
```suggestion
   community-spotlight-interviewing-guide
```

This is relative to the file.

**Review by ericholscher — APPROVED** [review]
Looks good with the above fix 👍

**Pull request merged**